src/pages/Faucet/index.tsx

```diff
@@ -61,9 +61,9 @@ export const FaucetPage = () => {
           Please make sure you are connected to Goerli network.
         </h2>
         <p className="p-1">
-          To connect to Goerli, within Metamask dropdown the Network tab at the
-          top and scroll and select Goerli test network (this will be there by
-          default).
+          To connect to the Goerli network, select your Metamask extension and
+          click the Network tab at the top. Select the Goerli test network (this
```

Comment:
mention that if they are not showing, enable test networks in metamask settings pls